homeassistant/components/switch_as_x/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "switch_as_x",
+  "name": "Switch as X",
+  "documentation": "https://www.home-assistant.io/integrations/switch_as_x",
+  "dependencies": ["switch"],
```

Comment:
We should set this up as an `after_dependencies`

Reply:
Fixed 👍